### ISSUE TITLE
New Field In SFDC

### ISSUE THREAD

**Issue by ckwittman** [mangoamelia/Balrog#3]
Hey Ameilia, 

Could you add in a Field in Salesforce: "Active (MT) Account" with a true/false indicator that I can edit on the account level for our customers?  

Thank you!

`Chris

**Comment by mangoamelia** [comment]
@ckwittman I believe we resolved this, correct?

**Comment by ckwittman** [comment]
Yes, I think the one piece, was being able to link the product subscription price to the account level?  were you able to get that to work?

`Chris

**Comment by mangoamelia** [comment]
Different ticket - this one is regarding the "Active MT User" field, which
I believe we resolved by adding the primary/secondary/billing contact/do
not contact field for contacts on an account, correct?

**Comment by ckwittman** [comment]
Oh yes, that is resolved.
